Makefile

```diff
@@ -120,11 +120,18 @@ clean:
 	rm -rf /tmp/avocado*
 	find . -name '*.pyc' -delete
 
-requirements:
+requirements-plugins:
+	for MAKEFILE in $(AVOCADO_PLUGINS);\
+		do AVOCADO_DIRNAME=$(AVOCADO_DIRNAME) make -C $$MAKEFILE requirements &>/dev/null && echo ">> INSTALL REQUIREMENTS-PLUGINS $$MAKEFILE" || echo ">> SKIP   $$MAKEFILE";\
```

Comment:
How about `DONE` or `PASS` instead? We don't actually know whether it failed because it's not a plugin, or whether it failed to install the requirements.